### DISCUSSION TITLE
Stale Action Not Appearing Under Actions

### DISCUSSION THREAD

**Discussion by glenn-jocher** [community/community#26168] [Actions]
<p>Hello, I’ve created a stale action file here:</p>
<p><a href="https://github.com/ultralytics/yolov3/blob/master/.github/workflows/stale.yml" rel="nofollow noopener">https://github.com/ultralytics/yolov3/blob/master/.github/workflows/stale.yml</a></p>
<p>But it does not appear anywhere under the actions tab, and does not seem to ever run. Action permissions 'Enable local and third party Actions for this repository ’ are enabled in setting. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1ce1e83abd86e7a215dd9e45c383a6d27170fc47.jpeg" data-download-href="/uploads/short-url/47vn7OF1RE3lj7PFLK3jUotkduv.jpg?dl=1" title="Screen Shot 2020-03-05 at 1.42.42 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26833433/181095903-3b0e6456-5249-42bf-b00d-68ae8ad4332f.jpg" alt="Screen Shot 2020-03-05 at 1.42.42 PM.png" data-base62-sha1="47vn7OF1RE3lj7PFLK3jUotkduv" width="629" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1ce1e83abd86e7a215dd9e45c383a6d27170fc47_2_629x499.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/1/1ce1e83abd86e7a215dd9e45c383a6d27170fc47_2_943x748.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/1ce1e83abd86e7a215dd9e45c383a6d27170fc47_2_1258x998.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1ce1e83abd86e7a215dd9e45c383a6d27170fc47_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-03-05 at 1.42.42 PM.png</span><span class="informations">2454×1950 415 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/glenn-jocher">@glenn-jocher</a> ,</p>
<p>Now the workflow exists on your actions tab, could you please confirm whether the issue is gone?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2e5596111be5fa5d31742e96790be0e8b9ed1839.png" data-download-href="/uploads/short-url/6BTkxehI95nOghsWjjg6XAz55sl.png?dl=1" title="scheduleworkflow.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181095905-db24af42-1909-4824-b065-3b3f5dcab2dd.png" alt="scheduleworkflow.png" data-base62-sha1="6BTkxehI95nOghsWjjg6XAz55sl" width="690" height="135" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/2e5596111be5fa5d31742e96790be0e8b9ed1839_2_690x135.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/2e5596111be5fa5d31742e96790be0e8b9ed1839_2_1035x202.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/2e5596111be5fa5d31742e96790be0e8b9ed1839.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2e5596111be5fa5d31742e96790be0e8b9ed1839_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">scheduleworkflow.png</span><span class="informations">1172×231 10.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Note: I forked your repo, and the workflow is successfully triggered on my side.</p>
<p>Thanks.</p>

**Comment by glenn-jocher** [comment]
<p>Yes thank you. It appears everything was workinig correctly, there was just a delay of a few hours from saving the yaml file to the action appearing under the actions tab.</p>
